typings.json

```diff
@@ -1,5 +1,7 @@
 {
   "ambientDependencies": {
+    "chai": "github:DefinitelyTyped/DefinitelyTyped/chai/chai.d.ts#a9680ac2efd3add0d38cd0331f30ad497d9dd1e0",
+    "mocha": "github:DefinitelyTyped/DefinitelyTyped/mocha/mocha.d.ts#a9680ac2efd3add0d38cd0331f30ad497d9dd1e0",
```

Comment:
You're probably right that our current typings setup would treat these Mocha function types as "global". Separating type definitions between client and test is feasible, but seems to be an overkill at this point.